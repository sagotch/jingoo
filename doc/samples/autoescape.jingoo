{%- set script = "<script></script>" -%}
{%- autoescape true -%}
{{ script }}
{% endautoescape -%}
{%- autoescape false -%}
{{ script }}
{% endautoescape -%}
