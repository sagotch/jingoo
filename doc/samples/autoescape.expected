&#60;script&#62;&#60;/script&#62;
<script></script>
